src/Zicht/Bundle/PageBundle/Admin/PageAdmin.php

```diff
@@ -160,7 +160,9 @@ public function configureFormFields(FormMapper $formMapper)
 
         if (($subject = $this->getSubject()) && $subject->getId()) {
             if ($subject->getContentItemMatrix() && $subject->getContentItemMatrix()->getTypes()) {
-                if (!property_exists($subject, 'contentItems')) {
+                // In newer versions of the Page entity, the contentItems property is private, hence we check
+                // for the existence of the getContentItems method
+                if (!(method_exists($subject, 'getContentItems') || property_exists($subject, 'contentItems'))) {
```

Comment:
Maybe a good point in time to create an interface?

Reply:
i think you can check if the subject is a instance of `Zicht\Bundle\PageBundle\Model\ContentItemContainer` (but that will make this fix breaking because most base pages implement those functions but not that interface.)